tests/tree_sitter.cpp

```diff
@@ -82,10 +90,21 @@ TEST_CASE("tree-sitter navigation", "[tree-sitter][.hide]") {
 TEST_CASE("tree-sitter print", "[tree-sitter][.hide]") {
     ts::Parser parser;
 
-    std::string source = "print(1+2)";
+    std::string source = "print(1+5)";
     ts::Tree tree = parser.parse_string(source);
     ts::Node root = tree.root_node();
+    
+    INFO(root.as_s_expr());
+    FAIL();
+}
 
+TEST_CASE("TestCase for program", "[tree-sitter]"){
```

Comment:
```suggestion
TEST_CASE("TestCase for program", "[tree-sitter][.hide]"){
```

or remove the `FAIL` below